docs/ReferenceArrayInput.md

```diff
@@ -328,6 +330,76 @@ Then to display a selector for the post tags, you should call `<ReferenceArrayIn
 <ReferenceArrayInput source="tags_ids" reference="tags" />
 ```
 
+## `readOnly`
+
+If `true`, the default input is in read-only mode and the user can't change the value.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏I think something like this could help users to understand the differences between `readOnly` and `disabled` props:
```suggestion
The `readOnly` prop set to true makes the element not mutable, meaning the user can not edit the control.
```